extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -86,6 +88,8 @@ require([
 			sourcePoint.initDetection();
 		}
 
+		pageFair.initDetection();
```

Comment:
I agree with nAndy. There is no point in loading external script if the module is disabled. However if you wrap this with if statement please add some logging somewhere why pagefair didn't start. Maybe you can add if statement inside `pagefair::init` method. Not sure which one is cleaner.